test/apps/draganddrop/compile.json

```diff
@@ -21,9 +21,5 @@
     }
   ],
 
-  "libraries": [
-    "../../framework",
```

Comment:
Maybe it makes sense not to hardcode the framework in the tests so that they can be used with other framework versions? Since you can `qx config set qx.library` in the testing code? But it's true that if not, the npm version will be tested, not the framework code in the repo, won't it, @johnspackman

Reply:
it's not necessary to explicitly set the framework location; testing is done by deployment, and deployment sets the global instance to the framework that is being deployed. If you're doing development on the framework, you presumably are also setting the global instance of the framework to your repo?